packages/scripts/src/lib/app-data/postProcess/assets.ts

```diff
@@ -65,52 +71,197 @@ export class AssetsPostProcessor {
       sourceAssetsFolders.unshift(parentAssetsFolder);
     }
 
-    // Generate a list of all deployment assets, merged across input sources
+    // Separate core assets and remote assets for separate processing
+    const coreAssetsHashmap: IContentsEntryHashmap = {};
+    const remoteAssetsBySourceFolder = new Map<string, IContentsEntryHashmap>();
+
     for (const sourceAssetsFolder of sourceAssetsFolders) {
       const sourceAssets = generateFolderFlatMap(sourceAssetsFolder, { includeLocalPath: true });
       const sourceAssetsFiltered = this.filterAppAssets(sourceAssets);
-      Object.entries(sourceAssetsFiltered).forEach(([relativePath, entry]) => {
-        if (mergedAssetsHashmap.hasOwnProperty(relativePath)) {
-          //  TODO - log override
+      const sourceMetadata = folderMetadata.get(sourceAssetsFolder);
+      const isSourceRemote = sourceMetadata?.remote === true;
+
+      if (isSourceRemote) {
+        // Track remote assets by source folder
+        if (!remoteAssetsBySourceFolder.has(sourceAssetsFolder)) {
+          remoteAssetsBySourceFolder.set(sourceAssetsFolder, {});
         }
-        mergedAssetsHashmap[relativePath] = entry;
-      });
+        const remoteAssets = remoteAssetsBySourceFolder.get(sourceAssetsFolder)!;
+        Object.assign(remoteAssets, sourceAssetsFiltered);
+      } else {
+        // Core assets - merge into coreAssetsHashmap (later folders can overwrite earlier ones)
+        Object.assign(coreAssetsHashmap, sourceAssetsFiltered);
+      }
+    }
```

Comment:
nit(blocking): The processing here feels messy, jumping in and out of custom logic for remote assets vs non. I'm trying to wrap my head around just how much of the processing is the same and how much is different? The code makes it imply like a lot of the processing is different, but in reality don't both core and remote assets get processed in the exact same way, just populating to different output folders? 

Once populated to folders a final `remote_sync` workflow could be used to keep the local remote_assets folder in sync with target remote server (just like local script copies deployment app_data/assets folder to app assets folder)

So I'd expect all things going smoothly this PR to actually only add a small handful of lines to this script and a small number to a new workflow step